pydarn/plotting/grid.py

```diff
@@ -155,20 +137,20 @@ def plot_grid(cls, dmap_data: List[dict], record: int = 0,
         # Find the record corresponding to the start time
         if start_time is not None:
             for record in range(len(dmap_data)):
-                dtime = dt.datetime(dmap_data[record]['start.year'],
+                date = dt.datetime(dmap_data[record]['start.year'],
                                     dmap_data[record]['start.month'],
                                     dmap_data[record]['start.day'],
                                     dmap_data[record]['start.hour'],
                                     dmap_data[record]['start.minute'])
-                time_diff = dtime - start_time
+                time_diff = date - start_time
                 if time_diff.seconds/60 <= time_delta:
                     break
             if time_diff.seconds/60 > time_delta:
                 raise plot_exceptions.NoDataFoundError(parameter,
                                                        start_time=start_time)
         else:
             record = 0
-            dtime = dt.datetime(dmap_data[record]['start.year'],
+            date = dt.datetime(dmap_data[record]['start.year'],
                                 dmap_data[record]['start.month'],
                                 dmap_data[record]['start.day'],
                                 dmap_data[record]['start.hour'],
```

Comment:
It's out of scope of the changes so can't add a suggestion but there's a stray dtime on line 163

Reply:
And on line 176

Reply:
Also in line 165 - 170, these options should be changed to **kwargs instead. Except the boundary = fov as there is a name change going on there and if we're setting it to a default it should be set, see an above comment for that.